app/models/zone.rb

```diff
@@ -35,6 +36,11 @@ class Zone < ApplicationRecord
   include AggregationMixin
   include ConfigurationManagementMixin
 
+  scope :visible, -> { where(:visible => true) }
+  default_value_for :visible, true
+
+  MAINTENANCE_ZONE_NAME = '__maintenance__'.freeze
```

Comment:
We don't wrap the default zone in double-underscores, so not sure we should do that here...I guess you're trying to avoid stepping on user defined zones?  If so, I'm fine with it ultimately, but if we decide to show it, it might be weird.

Reply:
@Fryguy yes, that's the reason - you discussed it with @agrare in this PR before. 
We are hiding this zone from UI and API, so it won't be visible.